lib/services/demoCampus.js

```diff
@@ -0,0 +1,78 @@
+const Schmervice = require('schmervice');
+
+module.exports = class DemoCampusService extends Schmervice.Service {
+    async insertDemoCampus(campus, phone_no) {
+        const { DemoCampus } = this.server.models()
+        try {
+            const demo = await DemoCampus
+                .query()
+                .insert({ campus, phone_no })
+            // return 'data inserted'
+            return demo;
+        } catch (error) {
+            console.log(error);
+            return 'Error is coming...'
+
+        }
+    }
+
+    async findall(limit, index) {
+
+        const { DemoCampus } = this.server.models();
+        try {
+            const demo = await DemoCampus
```

Comment:
read abut page and limit.
learn how to use them.
read about what is pagination.